app/views/folders/show.html.erb

```diff
@@ -1,16 +1,31 @@
-<h1><%= @current_folder.name %></h1>
+<h1><%= @folder.name %></h1>
+<p>Find me in app/views/folders/show.html.erb</p>
```

Comment:
This file needs to be changed back to the previous version that uses the partials.